packages/cli/src/commands/start.ts

```diff
@@ -11,11 +11,12 @@ import { readFile, writeFile } from "node:fs/promises";
 import { join } from "node:path";
 import process from "node:process";
 import * as p from "@clack/prompts";
+import * as picocolors from "picocolors";
```

Comment:
thank you for applying my comment

[nits]
we can use destructuring
It's not that important.
You can do it however is most comfortable for you.

Reply:
you are right. 
there is no consistance but I’ll keep it.
Thanks